terraform/aks/database.tf

```diff
@@ -0,0 +1,29 @@
+module "postgres" {
+  source = "./vendor/modules/aks//aks/postgres"
+
+  namespace                   = var.namespace
+  environment                 = var.environment
+  azure_resource_prefix       = var.azure_resource_prefix
+  service_name                = var.service_name
+  service_short               = var.service_short
+  config_short                = var.config_short
+  cluster_configuration_map   = module.cluster_data.configuration_map
+  use_azure                   = var.deploy_azure_backing_services
+  azure_enable_backup_storage = var.azure_enable_backup_storage
+}
+
+module "redis-cache" {
+  source                       = "./vendor/modules/aks//aks/redis"
+
+  namespace                    = var.namespace
+  environment                  = var.environment
+  azure_resource_prefix        = var.azure_resource_prefix
+  service_short                = var.service_short
+  config_short                 = var.config_short
+  service_name                 = var.service_name
+  cluster_configuration_map    = module.cluster_data.configuration_map
+  use_azure                    = var.deploy_azure_backing_services
+  azure_enable_monitoring      = var.enable_monitoring
+  azure_patch_schedule         = [{ "day_of_week": "Sunday", "start_hour_utc": 01 }]
+  azure_maxmemory_policy       = "allkeys-lfu"
+}
```

Comment:
there are 2 redis instances for tv. cache and queue
e.g.
teaching-vacancies-redis-cache-review-pr-6175
teaching-vacancies-redis-queue-review-pr-6175
the queue instance requires `maxmemory policy     : noeviction`

Reply:
the parameter has been changed as recommended